packages/forms/src/styled/common/SyledLegend.spec.tsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { render, screen } from 'garden-test-utils';
+import { StyledLegend } from './StyledLegend';
+
+describe('StyledLegend', () => {
```

Comment:
Only use this to test what is unique to `StyledLegend` apart from `StyledLabel`. i.e. `display: block;`, compact styling, or rendering as a `'LEGEND'` element.